src/eureka/S3_data_reduction/nirspec.py

```diff
@@ -320,11 +321,6 @@ def straighten_trace(data, meta, log):
                  mute=(not meta.verbose))
     meta.src_ypos = new_center
 
-    # update the median frame
```

Comment:
I don't think you want to delete these lines. We need to recalculate the median frame.

Reply:
The median frame gets calculated in optspex later, so no need to do this twice